### DISCUSSION TITLE
怎么解决育成出现StopIteration

### DISCUSSION THREAD

**Discussion by fg7om** [NateScarlet/auto-derby#138] [Q&A] [answered]
作者你好，请问怎么解决这种StopIteration的情况，具体是什么原因呢
![QQ图片20210808163505](https://user-images.githubusercontent.com/88612060/128626207-ddefa5e5-90c1-461e-b434-bbd6301ddf62.png)

**Comment by NateScarlet** [comment]
找不到角色详细按钮 
什么版本 last_screenshot 发一下

**Reply by fg7om** [reply]
下的应该是最新版本
![last_screenshot](https://user-images.githubusercontent.com/88612060/128626564-66c76f4f-454d-4996-a516-799ae132a33c.png)

**Reply by NateScarlet** [reply]
最新版本不叫版本 你这是想要我刻舟求剑
命令窗口标题有版本

**Comment by NateScarlet** [comment, marked as answer]
不支持汉化

Duplicated of #16

**Reply by fg7om** [reply]
是这样吗，我卸载汉化试试

**Reply by fg7om** [reply]
是真的，谢谢大佬
